frontend/src/components/AnnotateCard/index.jsx

```diff
@@ -97,6 +97,11 @@ function drawSkipButton(card, onSkip) {
     );
 }
 
+function promptAdjudicate(card, onSkip) {
+    let message = prompt("Please enter the reasons for skipping this card:");
```

Comment:
So I'm wondering if we could use a model for this instead?

https://react-bootstrap.github.io/components/modal/

It might make it easier to keep with the styling of the rest of the app. Otherwise my thoughts are these:

1. The reason should probably not be submittable blank. (would check on Wed with Ethan though)
2. My prompt says "localhost:800 says" which would confuse a normal user. Can we change this?
3. We'd want to change "ok" to "submit" since they are submitting a form